server/service/osquery_utils/queries.go

```diff
@@ -1351,6 +1352,71 @@ var SoftwareOverrideQueries = map[string]DetailQuery{
 			return mainSoftwareResults
 		},
 	},
+	// deb_last_opened_at collects last opened at information from DEB package files on Linux
+	// hosts. Joining this within the main software query is not performant enough to do on the
+	// device, so we do it on the server instead.
+	"deb_last_opened_at": {
+		Query: `
+		SELECT package, MAX(atime) AS last_opened_at
+		FROM deb_package_files 
+		CROSS JOIN file USING (path) 
+		WHERE type = 'regular' AND regex_match(file.mode, '[1357]', 0)
+		GROUP BY package
+	`,
+		Description:            "A software override query[^1] to append last_opened_at information to Linux DEB software entries.",
+		Platforms:              fleet.HostLinuxOSs,
```

Comment:
Guessing _not_ scoping this down to Debian distros is intentional, given `alien` et al (and same for RPM)?